modules/decline/src/Smithy4sCli.scala

```diff
@@ -111,24 +103,49 @@ class Smithy4sCli[Alg[_[_, _, _, _, _]], Op[_, _, _, _, _]](
         ).mapN { (input, entrypoint) =>
           val printers = entrypoint.printerApi
           val printer = printers.printer(endpoint)
-          printer.printInput(input) *>
+          val FO = printer.printInput(input) *>
             service
-              .asTransformation[GenLift[IO]#λ](entrypoint.interpreter)(
+              .asTransformation[GenLift[F]#λ](entrypoint.interpreter)(
                 endpoint.wrap(input)
               )
-              .flatMap(printer.printOutput)
-              .as(ExitCode.Success)
-              .recoverWith {
-                case e if endpoint.errorable.flatMap(_.liftError(e)).nonEmpty =>
-                  printer
-                    .printError(e)
-                    .as(ExitCode.Error)
-              }
+
+          FO.flatMap(printer.printOutput)
```

Comment:
Doesn't `printer.printOutput.flatMap` work? It should.

Reply:
It should but it wasnt compiling , Ill try to figure that out

Reply:
basically @kubukoz :

```
[info] [decline3] compiling 6 Scala sources to /Users/David.Francoeur/workspace/dev/smithy4s/modules/decline/target/jvm-3/classes ...
[error] [decline3] -- [E008] Not Found Error: /Users/David.Francoeur/workspace/dev/smithy4s/modules/decline/src/Smithy4sCli.scala:110:15 
[error] [decline3] 106 |            service
[error] [decline3] 107 |              .asTransformation[GenLift[F]#λ](entrypoint.interpreter)(
[error] [decline3] 108 |                endpoint.wrap(input)
[error] [decline3] 109 |              )
[error] [decline3] 110 |              .flatMap(printer.printOutput)
[error] [decline3]     |            ^
[error] [decline3]     |value flatMap is not a member of smithy4s.GenLift[F]#λ[I, E, O, endpoint.SI, endpoint.SO].
[error] [decline3]     |An extension method was tried, but could not be fully constructed:
[error] [decline3]     |
[error] [decline3]     |    cats.implicits.toFlatMapOps[
[error] [decline3]     |      ([SO] =>> smithy4s.GenLift[F]#λ[I, E, O, endpoint.SI, SO])
[error] [decline3]     |    , A](
[error] [decline3]     |      Smithy4sCli.this.service.asTransformation[
[error] [decline3]     |        smithy4s.GenLift[Smithy4sCli.this.F]#λ
[error] [decline3]     |      ](entrypoint.interpreter).apply[I, E, O, endpoint.SI, endpoint.SO](
[error] [decline3]     |        endpoint.wrap(input)
[error] [decline3]     |      )
[error] [decline3]     |    )(evidence$1)
[error] [decline3]     |
[error] [decline3]     |where:    F is a type in class Smithy4sCli with bounds <: [_] =>> Any
[error] [decline3] one error found
[error] [decline3] one error found
[error] [decline3] (decline3 / Compile / compileIncremental) Compilation failed
```

Reply:
exactly , i wasnt either . both ugly